docs/topics/web-console/assembly_asset-generation-ui.adoc

```diff
@@ -0,0 +1,53 @@
+= Generating assets for application deployment in {WebName}
+
+:context: asset-generation-ui
+
+[role=_abstract]
+Starting with 8.0.0, asset generation in {ProductShortName} {WebName} allows you to create different  assets (manifests) required to deploy applications from Cloud Foundry (CF) instances to {ocp-full} or Kubernetes. 
+
+To generate assets, you must become familiar with the following entities:
+
+* Archetypes - Archetypes contain one or more criteria tags to identify technologies such as EAP 6. An archetype helps you to classify applications based on technologies common to them. For asset generation, you can plan your migration paths by archetypes because you configure the other asset generation entities  in the archetype (for example, target profiles that contain the generators). 
+
+* Generators - A generator takes a templating engine, such as Helm, along with a set of templates as input configuration. Generators use the templates that are defined for a target platform and a migration path to generate deployment assets. When you run an asset generation process, the generator parses the application discovery manifest to create deployment assets.
++
+For example, the migration path for an EAP 6 application is EAP 8 and the target platform is Kubernetes. A generator can generate the required assets to migrate the EAP 6 application to EAP 8 that is to be deployed on Kubernetes. The assets are generated by parsing the discovery manifest of the EAP 6 application.
+
+* Target profiles - A target profile represents the target platform to which you want to migrate. It contains a seqeuence of generators. Through target profiles, you can run more than one generator, with each generator creating assets for a specific migration path. Thus, a target profile enables you to generate assets for multiple migration path-target platform combinations at the same time for a given Archetype. 
++
+For example, if you want to create assets to migrate an EAP 6 application as an OpenJDK 21 application _and_ as an EAP 8 application in {ocp-short} (target platform), you can achieve this through a target profile that has two generators, one for each migration path.
+
+* Application - The application includes the discovery manifest, tags, and the asset repository. 
++
+//provide the retrieve configuration link after publishing the doc
+The configuration retrieval process generates a discovery manifest that contains the platform and runtime configurations of an application deployed in the source platform (CF) instances. During asset generation, the discovery manifest is parsed by generator templates that produce the final assets for deployment. 
++
+The application also contains tags that you must configure as criteria tags in an Archetype.  
++
+The asset repository is where you find the generated assets for deployment in the target platform. You must configure Source Control credentials of the asset repository in an application before generating assets.
```

Comment:
You must configure is a little strong.
If not configured, it will fall-back to:
- the credentials associated with the source code repo
- the default creds for "source".
Not sure how you want to word it.